src/Core/Common/BresenhamLine/Code.cxx

```diff
@@ -44,9 +44,9 @@ void Vector()
   v[1] = 1;
   std::vector< itk::Offset<2> > offsets = line.BuildLine(v, 4);
 
-  for(unsigned int i = 0; i < offsets.size(); i++)
+  for(auto offset : offsets)
```

Comment:
Commit message has an unfinished sentence: `C++11 Range based for loops can be used in`.

Reply:
Done. Thanks.